### DISCUSSION TITLE
Add e-mail sendername

### DISCUSSION THREAD

**Discussion by Sibren** [umbraco/Umbraco-CMS#16101] [Features and ideas]
Right now, you can only specify the 'from' in Umbraco with an emailaddress, lets say '[mybeautifulemail@address.com](mailto:mybeautifulemail@address.com)'
Which means Gmail (and perhaps others) only show 'mybeautifulemail' as a name in your inbox and not a decent name.
I'd like to have the option to use a name too.

C# has this built in, but it seems Umbraco doesn't use it:
https://learn.microsoft.com/en-us/dotnet/api/system.net.mail.mailaddress?view=net-8.0

My proposition would be something like this:
![323222446-e9b262ec-0196-433b-a11f-33efae4f0b49](https://github.com/umbraco/Umbraco-CMS/assets/3531618/c1a8e2e3-d596-4bbe-bf25-b44affdb1f1a)

**Comment by skttl** [comment]
I believe you can type it as `My Beautiful Name <mybeautifulemail@address.com>` to get it that way. At least you can do like that with the `IEmailSender` interface.

**Comment by Sibren** [comment]
Hi Søren,

That seems to do the trick. However it might not always be very user
friendly.


Best regards,
Sibren

Op ma 22 apr 2024 om 08:11 schreef Søren Kottal ***@***.***>:
